### ISSUE TITLE
[Feature] Adjustable rectangle as screenoverlay for taking a screenshot

### ISSUE THREAD

**Issue by OliEfr** [greenshot/greenshot#310]
I noticed, that [Lightshot](https://app.prntscr.com/en/index.html) is creating a editable rectangle when creating a screenshot. So basically you draw your rectangle and after, you still can adjust the position and size of it. It would be nice to have such an option for greenshot as well.

Let me know if you are of the same opinion. Then I would start looking into the implementation.

![image](https://user-images.githubusercontent.com/31124624/114984979-8e046a80-9e92-11eb-8772-0b22e4bcf515.png)

**Comment by Lakritzator** [comment]
Yes, this is definitively something I like to see, with the possibility to have pre-defined sizes.
Unfortunately there are so many other places under construction.

I would have loved to refactor the capture form, so it would be easier to add stuff...
But if you want to have a go, you are more than welcome to.

Just let me know if you have questions!

And one of the challenges, we have a suboptimal process for translations, means when we need some additional settings in the UI, or need to write a text, this will not be easy to arrange the amount of translations we need.

**Comment by OliEfr** [comment]
Alright - I have taken a more closer look now into implementation and Im gonna be honest. It looks like much work. Unfortunately I wont be able to spend this much time right now.

Please go ahead and close this issue if you want. I'll keep using greenshot and if time allows it, I will come back to it.

**Comment by chaosink** [comment]
I'm also looking forward to this feature...
